.editorconfig

```diff
@@ -8,4 +8,4 @@ root = true
 [*.{java,sql}]
 charset = utf-8
 indent_style = space
-indent_size = 2
+indent_size = 4
```

Comment:
don't change indents, 2 space indent is the norm. While there are some files that were introduced by ODY that have 4sp indent, the standard is 2 space for CIRCE.

Reply:
Fixed